src/shared/lib/oql/generateParser.py

```diff
@@ -1,22 +0,0 @@
-#!/usr/bin/python
```

Comment:
why have you deleted this and rewritten the readme?

Reply:
Hi @adamabeshouse, `generateParser.py` was broke. Also, the part about `(function() {\n` was not necessary anymore (I think). So I thought I could simplify this process by removing `generateParser.py` and adding a readme.